packages/components/organisms/f-checkout/src/components/AgeVerification.vue

```diff
@@ -37,6 +37,18 @@
                 :value="selectedDate.year"
                 @input="selectionChanged($event, 'year')" />
 
+            <div
+                ref="AgeVerificationError"
+                tabindex="-1">
+                <error-message
```

Comment:
Can the ref and tabindex be straight on error-message component without the wrapping dev?

Reply:
Annoyingly no, If I have it on them, the screen reader tries to read the error icon rather than the error message

Reply:
Hmm, maybe we need to fix this in f-error-message so the icon is ignored by screen readers. But it is not a blocker for your PR, just annoying that we need a wrapper when it should work without it, right?

Reply:
Agreed, I'll add a ticket in